content/en/blog/2024/demo-skyscanner/index.md

```diff
@@ -0,0 +1,96 @@
+Skyscanner, we now have richer instrumentation and can rely on observability
+directly. As a result, we are ready to adopt a new
+[observability mindset](https://charity.wtf/2019/09/20/love-and-alerting-in-the-time-of-cholera-and-observability/),
+which requires training our engineers to work effectively with the new
+ecosystem. This allows them to react efficiently to any known or unknown issues,
+even under pressure.
+
+To achieve this, we believe that the best way to gain knowledge isn’t through
+one-time viewings of documents or videos. Instead, it’s through practical
+exercises that include situations with never-before-seen (or at least rarely
+seen) problems. This helps the company reduce the time to mitigate an issue
+(TTM), which goes from when a first responder acknowledges the incident to the
+time users stop suffering from the incident.
+
+## Environment
+
+To begin with, we need to set up an environment that demonstrates the best
+practices for monitoring and debugging using OpenTelemetry instrumentation and
+observability. For this, we propose the use of the official
+[OpenTelemetry Demo project](https://github.com/open-telemetry/opentelemetry-demo),
+which is a realistic example of a distributed system called Astronomy Shop.
+Thanks to its OpenTelemetry Protocol (OTLP), it allows us to simply point the
+standard OTLP exporter in the Collector to [New Relic](https://newrelic.com/),
+our chosen observability platform at Skyscanner which, like other platforms, is
+fully embracing open standards to ingest telemetry data.
+
+This system contains regressions that can be injected into the platform and
+helps us demonstrate the importance of Service Levels Objectives (SLOs),
+tracing, logs, metrics, etc. For instance, we can observe traffic flow through
+various components, as shown in the image below. Since part of the OpenTelemetry
+ecosystem is open-source, we can easily introduce any new features that will be
+reviewed by OpenTelemetry contributors.
+
+![Distributed tracing example in Astronomy shop](tracing_example.png)
+
+## Observability game day
+
+Once the environment is set up, we can introduce the Observability Game Day, an
+initiative based on the Wheel of Misfortune practices that Google uses and
+describes in the [Site Reliability Engineering book](https://sre.google/books/).
+
+This game simulates a production incident, where a moderator known as the game
+master (GM) conducts the session and someone from the audience spins the wheel
+and explains an incident or outage. The participants are then divided into teams
+and tasked with identifying and resolving the issue as quickly as possible. If
+the solution is not optimal, the GM can help by introducing a new tool or view,
+which gives a different perspective on how to tackle the incident (knowledge
+sharing). This exercise can be repeated multiple times for different incidents.
+
+![Wheel of misfortune example](wheel.png)
```

Comment:
did you create this image or is it taken out of the book?

Reply:
IIRC, I created it using an online tool where I was able to add the desired text :)

Reply:
ok, that's all I wanted to know